src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to order the properties within each selector in a specific order: positioning, box model, typography, visual, misc. In this case, 'height' and 'width' should come before 'background-image'.